clients/hadoop-to-venice-bridge/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -2513,6 +2526,8 @@ protected void setupDefaultJobConf(
        */
       conf.set(KAFKA_INPUT_TOPIC, pushJobSetting.kafkaInputTopic);
       conf.set(KAFKA_INPUT_BROKER_URL, pushJobSetting.kafkaInputBrokerUrl);
+      conf.setLong(REPUSH_TTL_IN_HOURS_OVERRIDE, pushJobSetting.repushTtlInHours);
+      conf.setInt(REPUSH_TTL_POLICY, 0); // only support one policy thus not allow any value passed in.
```

Comment:
Can we use `String` for setting policy?

Reply:
I think we can but I'd like to follow our convention here that using int to specify the policy, such as `BufferReplayPolicy`, `BatchConflictResolutionPolicy`

Reply:
We should at-least use `SOME_DEFAULT_POLICY.value()` instead of `0` to make it easy to read.